### DISCUSSION TITLE
Is there any way to mass update the cert a host is using?

### DISCUSSION THREAD

**Discussion by clstr** [NginxProxyManager/nginx-proxy-manager#3013] [Q&A]
I use a custom wildcard cert and when it is time for renewal i have to go in add the import the cert and replace it on each host. Is there a way to do this more simply via the webui?

**Comment by ImChiip** [comment]
up on this same here

**Comment by shalafi99** [comment]
I do not think there is (a way to do it using the webui). Using the suggestion given in #2317 as a workaround.
